lib/internal/test_runner/test.js

```diff
@@ -691,4 +691,13 @@ class Suite extends Test {
   }
 }
 
-module.exports = { kCancelledByParent, kDefaultIndent, kSubtestsFailed, kTestCodeFailure, Test, Suite, ItTest };
+module.exports = {
+  kCancelledByParent,
+  kDefaultIndent,
+  kSubtestsFailed,
+  kTestCodeFailure,
+  kHookFailure,
+  Test,
+  Suite,
+  ItTest,
```

Comment:
nit: alphabetical order
```suggestion
  ItTest,
  kCancelledByParent,
  kDefaultIndent,
  kHookFailure,
  kSubtestsFailed,
  kTestCodeFailure,
  Suite,
  Test,
```